src/Components/Aspire.OpenAI/README.md

```diff
@@ -0,0 +1,123 @@
+# Aspire.OpenAI library
+
+Registers [OpenAIClient](https://github.com/openai/openai-dotnet?tab=readme-ov-file#using-the-openaiclient-class) as a singleton in the DI container for using the OpenAI REST API. Enables corresponding metrics, logging and telemetry.
+
+## Getting started
+
+### Prerequisites
+
+- An OpenAI REST API compatible service like OpenAI.com, Azure AI OpenAI, ollama.com and others.
+- A API key.
+
+### Install the package
+
+Install the .NET Aspire OpenAI library with [NuGet](https://www.nuget.org):
+
+```dotnetcli
+dotnet add package Aspire.OpenAI
+```
+
+## Usage example
+
+In the _Program.cs_ file of your project, call the `AddOpenAIClient` extension method to register an `OpenAIClient` for use via the dependency injection container. The method takes a connection name parameter.
+
+```csharp
+builder.AddOpenAIClient("openaiConnectionName");
+```
+
+You can then retrieve the `OpenAIClient` instance using dependency injection. For example, to retrieve the client from a Web API controller:
+
+```csharp
+private readonly OpenAIClient _client;
+
+public ChatController(OpenAIClient client)
+{
+    _client = client;
+}
+```
+
+To learn how to use the `OpenAIClient` class refer to [Using the OpenAIClient class](https://github.com/openai/openai-dotnet?tab=readme-ov-file#using-the-openaiclient-class).
```

Comment:
In Azure SDK terminology, we don't typically refer to clients as classes -- it is not technically incorrect to do so, but this sounds strange to my ears.  It might make sense to reword this to something like "learn how to use the `OpenAI` client library" ?

@AngelosP, can you recommend an official document the Aspire documents could link to for helping Aspire users onboard to the client?  @scottaddie, FYI that this may be an opportunity to build out docs about SCM-based clients that the Aspire team could leverage going forward.